src/igvfd/upgrade/donor.py

```diff
@@ -41,3 +41,11 @@ def donor_3_4(value, system):
     if 'ethnicity' in value:
         value['ethnicities'] = value['ethnicity']
         del value['ethnicity']
+
+
+@upgrade_step('human_donor', '4', '5')
+@upgrade_step('rodent_donor', '2', '3')
+def donor_2_3_4_5(value, system):
```

Comment:
not sure I like the 2_3_4_5 I am not saying i know how to call it, but 2_3 would probably be better than 2_3_4_5

Reply:
Please let me know what name you would like?  2_3 is already used in a prior upgrade step.

Reply:
3_4

Reply:
Resolved with changes for version for rebase.